CONTRIBUTING.md

```diff
@@ -170,6 +193,13 @@ Even if some of these dependencies are not in active use, the git "update" comma
     url = https://github.com/nhabuiduc/TypescriptSyntaxPaste.git
 ```
 
+#### Issues Compiling Proto Input
+Proto Input appears to be very sensitive to its development environment. If you are opening ProtoInput.sln in Visual Studio 2022, you may be prompted to uprade the project, but doing this might complicate things. If you cannot get it to compile, try opening a fresh version of ProtoInput.sln (so you see the ugprade prompt again) and make sure that all options are set to "No Upgrade". Upgrading can potentially lead to syntax errors in the compiling process. If you see a FILE_INFO_BY_HANDLE_CLASS syntax error pop up numerous times during compiling, this is a likely culprit.

```

Comment:
```suggestion
Proto Input appears to be very sensitive to its development environment. If you are opening ProtoInput.sln in Visual Studio 2022, you may be prompted to upgrade the project, but doing this might complicate things. If you cannot get it to compile, try opening a fresh version of ProtoInput.sln (so you see the upgrade prompt again) and make sure that all options are set to "No Upgrade". Upgrading can potentially lead to syntax errors in the compiling process. If you see a `FILE_INFO_BY_HANDLE_CLASS` syntax error pop up numerous times during compiling, this is a likely culprit.
```